### DISCUSSION TITLE
Installation of Finance module for local development environment

### DISCUSSION THREAD

**Discussion by mohsinansarirepo** [egovernments/Digit-Core#83] [Public Finance]
We have tried "Quickstart" on local machine to get familiar with the installation steps. Now, we are trying to install finance module by cloning the "master" repository and following the similar steps as followed in quickstart but after installation pods are not created. 

Please guide us what we are missing or there is some other installation method which we have to follow.

**Comment by mohsinansarirepo** [comment]
With "Quickstart" superuser login we are getting the finance module but when any of the side menu is clicked we get "undefined-uat.local.digit’s server IP address could not be found". From which file/api the server name can be configured ?

**Comment by Gowtham-egov** [comment]
Hi Mohsin. If you want to install any module such as finance, you can use the following repo https://github.com/egovernments/DIGIT-DevOps/tree/optimal-tf-changes by using go script

**Comment by mohsinansarirepo** [comment]
Hi @Gowtham-egov ,
I tried above mentioned repository with the "standalone_installer.go" script but finance service pod id giving "CreateContainerConfigError" error. I have also changed the git SSH key which was changed by git.
Further I am trying to create local development environment of finance module on my high configuration machine, so am I facing this issue due to the reason finance module can be installed on cloud only ? 

![image](https://github.com/egovernments/Digit-Core/assets/92909304/46e8383b-1e32-4e4e-a49b-5fc0989f116d)

**Comment by Gowtham-egov** [comment]
Can you share with me the kubeconfig for your cluster? I will check from my
end.

**Comment by mohsinansarirepo** [comment]
Please let me know if you need anything else. I can provide access of anydesk of my system.

[myk3dconfig.TXT](https://github.com/egovernments/Digit-Core/files/11564085/myk3dconfig.TXT)

**Comment by Gowtham-egov** [comment]
Hi mohsin. share me the logs of the pod. It is enough.

**Comment by mohsinansarirepo** [comment]
[egov-finance-log.TXT](https://github.com/egovernments/Digit-Core/files/11565168/egov-finance-log.TXT)

**Comment by mohsinansarirepo** [comment]
Also below services are not running
1. egov-filestore-5bbbc76658-s28zt
2. egov-persister-5d575d4f9c-llrql
3. egov-document-uploader-76c8885-lpnzt
4. egov-mdms-service-75fbb65597-fcdtr
5. egov-localization-87766fc74-xblx5
6.  egov-searcher-66b7cfb5d9-5lz9p
7.  egov-url-shortening-69548b667b-rx8tt
8. billing-service-5b9d5ffbb9-gptts
9. egov-location-5f475588fd-qljjt
10 . collection-services-7547d487b7-f7qwl

**Comment by Gowtham-egov** [comment]
![image](https://github.com/egovernments/Digit-Core/assets/112694085/caa01264-4906-4550-b15b-49dcf8898a98)

**Reply by Gowtham-egov** [reply]
Have you updated this IP? PostgreSQL driver class is not compatible with the given connection URL. Have you created cluster in aws?

**Reply by mohsinansarirepo** [reply]
No. we have created clusters in local Linux machine.

**Reply by Gowtham-egov** [reply]
Hi mohsin. I'm not getting what was the issue exactly. Can we connect for a while?

**Comment by mohsinansarirepo** [comment]
Sure @Gowtham-egov we can connect anytime after 6:00 PM as right now I am attending the training of UPYOG by eGov. 

Also, please tell the working github repository of finance module because after resolving issues and deploying the finance module successfully on wildfly i am not able to access it on browser. Please check https://github.com/egovernments/Digit-Core/discussions/89   for the more details.
